### PR TITLE
v2v: Compose image name dynamically

### PR DESCRIPTION
**Pull request by mareklibra**
Allows downstream patches.

### CODE REVIEW

**Comment by suomiy** [comment]
prettier complains..

File: src/config/index.js [hunk 1]
```diff
@@ -1,4 +1,19 @@
+import { get } from 'lodash';
+
+const getRegistry = () => {
+  const registry = get(window, 'SERVER_FLAGS.registry');
+  if (!registry) {
```
Comment (suomiy) [inline src/config/index.js]: we should not forget to setup SERVER_FLAGS for tests and cosmos next time. Or alternatively set a default value here.

File: src/config/index.js [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    throw new Error('kubevirt: window.SERVER_FLAGS.registry is not set!');
+  }
+  return registry;
+};
+
+const getRegistryV2v = () => get(getRegistry(), 'v2v.url');
+const getImageTagConversion = () => get(getRegistry(), 'v2v.conversionTag');
```
Comment (suomiy) [inline src/config/index.js]: image tags could have a default value latest

**Comment by mareklibra** [comment]
Info: Working on better approach via ConfigMap

**Comment by mareklibra** [comment]
Finished, @rawagner, @suomiy , can you please have a look?

**Pull request merged**